install-application/action.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'install-application'
+description: 'Power Platform Install Application'
+inputs:
+  environment-url:
+    description: 'URL of Power Platform environment; e.g. "https://test-env.crm.dynamics.com".'
+    required: true
+
+  environment-id:
+    description: 'Target environment application will be installed to.'
```

Comment:
as discussed earlier this week: we should wait with this new action until we have the proper pac CLI support to allow the user to specify a --environment parameter/property (which could be either a guid or url)